doc/source/user/how-to-how-to.rst

```diff
@@ -0,0 +1,114 @@
+.. _how-to-how-to:
+
+##############################################################################
+How to write a NumPy how-to
+##############################################################################
+
+How-tos get straight to the point -- they
+
+  - answer a focused question, or
+  - narrow a broad question into focused questions that the user can
+    choose among.
+
+******************************************************************************
+A stranger has asked for directions...
+******************************************************************************
+
+**"I need to fill my tank."**

```

Comment:
This is a non-sequitur. If a stranger asks for directions, there should be a destination in the question. It is not clear they are in a car, looking for petrol.